### ISSUE TITLE
Add spec.muteTimeIntervals[].timeIntervals[].location to AlertmanagerConfig

### ISSUE THREAD

**Issue by sathieu** [prometheus-operator/prometheus-operator#5512]
**What is missing?**

https://github.com/prometheus-operator/prometheus-operator/blob/27943fe43862d313a63b518b6ee67b59d7680c1d/pkg/apis/monitoring/v1beta1/alertmanager_config_types.go#L944-L960

is missing [location](https://prometheus.io/docs/alerting/latest/configuration/#time_interval)

**Why do we need it?**

Not everybody uses UTC ;-)!

**Comment by slashpai** [comment]
I see `location` was added only in v0.25 and added location field validation in https://github.com/prometheus-operator/prometheus-operator/pull/5263/files. It needs to be supported in CRD though :)

**Comment by chiukapoor** [comment]
Hello Team, Is this issue still active? If not, may I work on it?

@mcbenjemaa @slashpai

**Comment by mcbenjemaa** [comment]
Yes, i will do it. When have free time

**Comment by slashpai** [comment]
> Hello Team, Is this issue still active? If not, may I work on it?
> 
> @mcbenjemaa @slashpai

Yes. PR is already there https://github.com/prometheus-operator/prometheus-operator/pull/5518 just needs to address comments and add tests :)

**Comment by LS80** [comment]
This is still needed.

**Comment by mkozikowsk** [comment]
Is there any update on this?

**Comment by 4devnull** [comment]
Hi,
This is still needed :) 
Possible add "location" spec ?
